### DISCUSSION TITLE
Database Data

### DISCUSSION THREAD

**Discussion by apassan23** [joshidipesh12/rss-swift#3] [Q&A]
What should be the data to be stored in database?

**Comment by apassan23** [comment]
Some Suggestions:

- Links to RSS Feeds for a particular user
- Read and unread posts (Maybe post ids?)

**Reply by joshidipesh12** [reply]
Yeah 👍
In 1st one can put rss links, added by user named YOUR FEEDS.

**Reply by joshidipesh12** [reply]
And all articles will be unread by default?

**Reply by apassan23** [reply]
Yes all posts will unread by default.
